examples/acid2.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
```

Comment:
The acid2 test got added last week already, under https://github.com/niklasvh/html2canvas/blob/master/tests/cases/acid2.html so I guess this one isn't necessary.